Sources/ClientRuntime/Networking/Http/URLSession/URLSessionConfiguration+HTTPClientConfiguration.swift

```diff
@@ -14,6 +14,9 @@ extension URLSessionConfiguration {
     public static func from(httpClientConfiguration: HttpClientConfiguration) -> URLSessionConfiguration {
         let config = URLSessionConfiguration.default
         config.timeoutIntervalForRequest = httpClientConfiguration.socketTimeout
+        config.httpShouldSetCookies = false
+        config.httpCookieAcceptPolicy = .never
+        config.httpCookieStorage = nil
```

Comment:
Not strictly related to the rest of this PR but this should be done... the URLSession is configured to ignore cookies completely.

AWS HTTP APIs should never use cookies, but this guards against any weirdness from a proxy or gateway a customer may use.

(lol been bit by this before on other apps, hence I'm adding it here)